tests/PHPStan/Tests/TypeInferenceTestCaseTest.php

```diff
@@ -0,0 +1,59 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Tests;
```

Comment:
This namespace could be `Testing`.

Reply:
already existing classes in the same folder use `PHPStan\Tests`.

`Testing` is used in the `src/` folder

Reply:
But here you're testing **PHPStan\Testing\TypeInferenceTestCase** class.